CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 Changelog
 =========
 
+0.2.0 "Khan"
+------------
+**Date:** 2019-02-XX
```

Comment:
don't forget this :)

Reply:
Yes the date is currently not clear but very soon it is.

Reply:
I will set the release date to next Monday 2019-02-10